staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1/types.go

```diff
@@ -93,6 +93,32 @@ type APIServiceSpec struct {
 
 	// leaving this here so everyone remembers why proto index 6 is skipped
 	// Priority int64 `json:"priority" protobuf:"varint,6,opt,name=priority"`
+
+	// ScopedResources declares a range of scoped resources to tell what resources to provide by the APIService. there can be more resources provided
+	// by the servicebackend. k8s aggregator focuses on the resources declared by ScopedResources, if it is defined.
+	ScopedResources []ScopedResource `json:"scopedResources,omitempty" protobuf:"bytes,7,opt,name=scopedResources"`
+}
+
+type ResourceScope string
+
+const (
+	ResourceScopeCluster   ResourceScope = "Cluster"
+	ResourceScopeNamespace ResourceScope = "Namespace"
+)
+
+type ScopedResource struct {
+	ResourceScope ResourceScope `json:"resourceScope" protobuf:"bytes,2,name=resourceScope"`
```

Comment:
the protobuf number is not right. there are 2 `2`s. please correct them. the rest is good.

Reply:
Done.